ext/standard/url.c

```diff
@@ -19,6 +19,10 @@
 #include <ctype.h>
 #include <sys/types.h>
 
+#ifdef __SSE2__
```

Comment:
@laruence it seems this macro does not work with MSVC compiler, see https://stackoverflow.com/questions/18563978/detect-the-availability-of-sse-sse2-instruction-set-in-visual-studio , any plan to add it to be consistent?